Sources/TuistGraph/DependenciesGraph/DependenciesGraph.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import TSCBasic
+
+/// A directed acyclic graph (DAG) that Tuist uses to represent the third party dependency tree.
+public struct DependenciesGraph: Equatable, Codable {
+    /// A dictionary where the keys are the names of dependencies,
+    /// and the values are the dependencies itself.
+    public var nodes: [String: DependenciesGraphNode]
```

Comment:
I'd rename this to `thirdPartyDependencies`. `nodes` is IMHO not descriptive enough and for the actual DAG representation it will be necessary to create a new property that will be one-to-many (`[DependenciesGraphNode: [DependenciesGraphNode]]`)

Reply:
It makes sense to me.

I have expected some "graph" related feedback. Thank you :)